projects/ad7768-1fmcz/src/ad77681evb.c

```diff
@@ -81,7 +81,7 @@ struct spi_engine_init_param spi_eng_init_param  = {
 	.spi_engine_baseaddr = AD77681_SPI1_ENGINE_BASEADDR,
 	.cs_delay = 0,
 	.data_width = 32,
-	.ref_clk_hz = AD77681_SPI_ENG_REF_CLK_FREQ_HZ,
+	.ref_clk_hz = 80000000,
```

Comment:
Isn't AD77681_SPI_ENG_REF_CLK_FREQ_HZ anyway 80 MHz (https://github.com/analogdevicesinc/hdl/pull/591)?

Reply:
https://github.com/analogdevicesinc/no-OS/blob/7980ad177a0d15849116d01417318c1730bf82f7/projects/ad7768-1fmcz/src/ad77681evb.c#L64

The value inside `xparameters.h`:

![image](https://user-images.githubusercontent.com/26061529/96427157-e0780d80-1206-11eb-8cd9-e99efe8dc733.png)